automated install/basic-install.sh

```diff
@@ -2442,15 +2442,15 @@ FTLcheckUpdate() {
         if [[ ${ftlLoc} ]]; then
             local FTLversion
             FTLversion=$(/usr/bin/pihole-FTL tag)
+
             local FTLlatesttag
+            FTLlatesttag=$(curl -s https://api.github.com/repos/pi-hole/FTL/releases/latest | jq -r .tag_name)
 
-            if ! FTLlatesttag=$(curl -sI https://github.com/pi-hole/FTL/releases/latest | grep --color=never -i Location: | awk -F / '{print $NF}' | tr -d '[:cntrl:]'); then
+            if [ -z "${FTLlatesttag}" ]; then
                 # There was an issue while retrieving the latest version
                 printf "  %b Failed to retrieve latest FTL release metadata" "${CROSS}"
                 return 3
-            fi
-
-            if [[ "${FTLversion}" != "${FTLlatesttag}" ]]; then
+            elif [[ "${FTLversion}" != "${FTLlatesttag}" ]]; then
```

Comment:
You could re-use the `version_converter` we have in PADD to really check if the new version is bigger

https://github.com/pi-hole/PADD/blob/a9681264c83529a2bd826bc86dd933d85b9b15bf/padd.sh#L1078-L1082

Reply:
Included on the new code.